packs/zabbix/actions/lib/base.py

```diff
@@ -0,0 +1,32 @@
+from pyzabbix import ZabbixAPI
+
+__all__ = [
+    'BaseZabbixAction'
+]
+
+
+class Action(object):
```

Comment:
Should import base Action from st2common like https://github.com/StackStorm/st2contrib/blob/master/packs/aws/actions/lib/action.py#L9.